iib/workers/tasks/opm_operations.py

```diff
@@ -190,3 +191,217 @@ def _serve_cmd_at_port(serve_cmd, cwd, port, max_tries, wait_time):
         rpc_proc.kill()
 
     raise IIBError(f'Index registry has not been initialized after {max_tries} tries')
+
+
+def opm_migrate(index_db, base_dir):
+    """
+    Migrate SQLite database to File-Based catalog using opm command.
+
+    :param str index_db: path to SQLite index.db which should migrated to FBC.
+    :param str base_dir: base directory to which the database file should be saved.
```

Comment:
How about 'base directory where catalog should be created'?

Reply:
Thank you! Better that mine your English is. Yes, hrrrm.